### DISCUSSION TITLE
Support for Vue3

### DISCUSSION THREAD

**Discussion by mpgalaxy** [buefy/buefy#3861] [Q&A] [answered]
Hi, I saw the Issue regarding support for Vue3 and I wanted to add somthing to this:

**Someone did port buefy to Vue3** ( https://github.com/kikuomax/buefy ),
maybe the core devs could help the author to maintain this, 99% of the work is already done! :)

This would be a win win for all.
@the-core-devs-of-buefy:  **Please consider this (!) :)**

**Comment by wesdevpro** [comment, marked as answer]
Support for vue3 is currently in progress 😎

 You can find the latest version of Buefy-next here https://github.com/ntohq/buefy-next
